Command/RotateEncryptionProfileCommand.php

```diff
@@ -48,23 +48,23 @@ protected function configure()
                 null
             )
             ->addOption(
-                'start',
+                'offset',
                 null,
-                InputOption::OPTIONAL,
+                InputOption::VALUE_OPTIONAL,
                 'Starting record',
                 0
             )
             ->addOption(
                 'limit',
                 null,
-                InputOption::OPTIONAL,
+                InputOption::VALUE_OPTIONAL,
```

Comment:
the option options are a bit misleading which is why this can be confused even by people who have done this alot. An option is itself already optional always no matter how you configure it. What VALUE_OPTIONAL causes here is that it makes it so that when the option is passed you wont have to put a value ie without VALUE_OPTIONAL you'll have to pass `--limit={some limit}` with VALUE_OPTIONAL it lets you pass `--limit` without a value at all. For this option i dont think you want that